### PR TITLE
Address error in water consumption calculation in evap cooled Chiller:Electric:EIR

### LINKED ISSUE #7951: Error in water consumption calculation in evaporatively cooled Chiller:Electric:EIR
Issue overview
--------------
Model of evaporatively cooled Chiller:Electric:EIR incorrectly uses air density to calculate volumetric water consumption rate, which results in 1000-fold errors in calculated chiller water use (line 2189):

```cpp
((this->CondOutletHumRat - DataLoopNode::Node(this->CondInletNodeNum).HumRat) * this->CondMassFlowRate) / RhoAir;
```
Water density should rather be used instead.

### Details
Some additional details for this issue (if relevant):
 - Windows 10
 - EnergyPlus 8.9

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by Nigusse**
Pull request overview
---------------------
 - Fixes #7951
 - Fixes evaporatively cooled Chiller:Electric:EIR incorrect water consumption rate calculation. Replaced air density with water density in converting water consumption mass flow rate into volume flow rate.
 - Diffs are expected for these two output variables: *Chiller Evaporative Condenser Mains Supply Water Volume* and *Chiller Evaporative Condenser Water Volume*.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [x] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [x] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [x] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [x] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [x] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by Nigusse** [comment]
Evaporatively cooled Electric Chiller EIR condenser water consumption [volume, m3] comparison between develop and this branch. The fix replaced air density with water density in *EvaporativelyCooled* condenser water consumption calculation. See the before and after the fix water consumption plots.

EvapCooledCondWaterConsum

File: src/EnergyPlus/ChillerElectricEIR.cc [hunk 1]
```diff
@@ -2183,11 +2183,10 @@ namespace ChillerElectricEIR {
             }
 
             if (this->CondenserType == EvapCooled) {
-                Real64 RhoAir = Psychrometrics::PsyRhoAirFnPbTdbW(
-                    DataEnvironment::StdBaroPress, condInletTemp, DataLoopNode::Node(this->CondInletNodeNum).HumRat, RoutineName);
+                Real64 const RhoWater = Psychrometrics::RhoH2O(DataGlobals::InitConvTemp);
```
Comment (Nigusse) [inline src/EnergyPlus/ChillerElectricEIR.cc:2157]: Replaced moist air density with water density (at 5.05 deg C). Air density should not have been used.

File: src/EnergyPlus/ChillerElectricEIR.cc [hunk 2]
```diff
@@ -2183,11 +2183,10 @@ namespace ChillerElectricEIR {
...
                 // CondMassFlowRate is already multiplied by PLR, convert to water use rate
                 this->EvapWaterConsumpRate =
-                    ((this->CondOutletHumRat - DataLoopNode::Node(this->CondInletNodeNum).HumRat) * this->CondMassFlowRate) / RhoAir;
+                    ((this->CondOutletHumRat - DataLoopNode::Node(this->CondInletNodeNum).HumRat) * this->CondMassFlowRate) / RhoWater;
```
Comment (Nigusse) [inline src/EnergyPlus/ChillerElectricEIR.cc:2160]: Variable renamed to reflect it is water.

**Comment by Nigusse** [comment]
*Chiller Evaporative Condenser Water Volume* and *Chiller Evaporative Condenser Mains Supply Water Volume* output variables for Chiller:Electric:EIR chiller type are expected to show diffs because of this fix but non of the test suite files have them.

**Comment by Nigusse** [comment]
This pull request is ready for review.

**Comment by mitchute** [comment]
> _Chiller Evaporative Condenser Water Volume_ and _Chiller Evaporative Condenser Mains Supply Water Volume_ output variables for Chiller:Electric:EIR chiller type are expected to show diffs because of this fix but non of the test suite files have them.

Should we add a file to the test suite to address this?

**Comment by mjwitte** [comment]
> Chiller Evaporative Condenser Water Volume and Chiller Evaporative Condenser Mains Supply Water Volume output variables for Chiller:Electric:EIR chiller type are expected to show diffs because of this fix but non of the test suite files have them.

> Should we add a file to the test suite to address this?
@mitchute @Nigusse Assuming there's an existing file(s) that has a chiller with evaporative condenser, I would add these (and any other pertinent) output variables there. If there is no file with an evaporative condenser chiller, then yes, we should add one or more or just modify one of the many files with a chiller in them.

**Comment by mitchute** [comment]
> > > Chiller Evaporative Condenser Water Volume and Chiller Evaporative Condenser Mains Supply Water Volume output variables for Chiller:Electric:EIR chiller type are expected to show diffs because of this fix but non of the test suite files have them.
> >
> > Should we add a file to the test suite to address this?
>  @mitchute @Nigusse Assuming there's an existing file(s) that has a chiller with evaporative condenser, I would add these (and any other pertinent) output variables there. If there is no file with an evaporative condenser chiller, then yes, we should add one or more or just modify one of the many files with a chiller in them.

@mjwitte There are no files with an evaporatively cooled condenser. @Nigusse could you add a new file, or modify one of the existing ones?

**Comment by Nigusse** [comment]
OK. I will create one by modifying existing example file.

**Comment by mitchute** [comment]
No point in waiting any longer on this. Merging.

**Pull request merged**